src/ts/uimanager.ts

```diff
@@ -111,6 +113,7 @@ export class UIManager {
   private config: InternalUIConfig; // Conjunction of provided uiConfig and sourceConfig from the player
   private managerPlayerWrapper: PlayerWrapper;
   private focusVisibilityTracker: FocusVisibilityTracker;
+  private _subtitleSettingsManager: SubtitleSettingsManager;
```

Comment:
nit: would be nice to follow the same pattern we have for `config`, for example

Reply:
[Replace get accessor with regular method](https://github.com/bitmovin/bitmovin-player-ui/pull/615/commits/c5d5e4f8000621dd6bb1141d45aa557782b83c00)